src/client/shuttle.js

```diff
@@ -29,7 +31,8 @@ export class Shuttle extends Moving {
     this.actions.start(stage)
 
     // Fire a bullet on the 'fire' action
-    this.actions.on('fire', when => this.fireBullet(stage, when))
+    this.actions.on('fire', () => this.fire())
```

Comment:
```suggestion
    this.actions.on('fire', () => this.cannon.fire.emit(this))
```

And then :fire: the `fire()` method